src/index.html

```diff
@@ -7,6 +7,44 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+
+    <div class="calendar calendar--day-fri calendar--day-month-30">
```

Comment:
Read the task carefully - you need to rename your modifiers.

Also checklist #2